conda-recipes/iqsharp/ToolStandalone.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
why do we need a different project? can you just target the existing one?

Reply:
`dotnet publish` has a lot of bugs when running on a project that has `<PackAsTool>true</PackAsTool>`. This was the most reliable workaround that I could find for those issues.